experimenter/experimenter/experiments/constants.py

```diff
@@ -371,7 +371,13 @@ class EmailType(models.TextChoices):
     }
 
     FEATURE_ENABLED_MIN_UNSUPPORTED_VERSION = Version.FIREFOX_104
-    DESKTOP_ROLLOUT_MIN_SUPPORTED_VERSION = Version.FIREFOX_114
+    ROLLOUT_MIN_SUPPORTED_VERSION = {
+        Application.DESKTOP: Version.FIREFOX_115,
+        Application.FENIX: Version.FIREFOX_116,
+        Application.FOCUS_ANDROID: Version.FIREFOX_116,
+        Application.IOS: Version.FIREFOX_116,
+        Application.FOCUS_IOS: Version.FIREFOX_116,
+    }
 
     ROLLOUT_SUPPORT_VERSION = {
```

Comment:
I think we can get rid of this variable now if we're defining the min for all applications in `ROLLOUT_MIN_SUPPORTED_VERSION`

Reply:
Note to self: we'll need to update the docs for this https://experimenter.info/deep-dives/experimenter/rollouts#supported-platforms-and-minimum-version-targeting

Reply:
I don't think we actually want to remove that as afaik that's referring to when the overall concept of rollouts existed for those apps. That check actually raises a ValidationError. I think what might make more sense is to change the name of `ROLLOUT_MIN_SUPPORTED_VERSION` to something like `ROLLOUT_LIVE_RESIZE_MIN_SUPPORTED_VERSION`?

Reply:
@eliserichards oh yeah it's easy to think of that case, ie someone just goes right ahead and launches a 100% rollout and never intends to change it.  I think that pattern is somewhat common.  I think we should stick with the pattern we had, where if you launch a rollout before the re-enrollment supported version we just warn that they won't be able to effectively resize it.

Reply:
Yeah and I think the big thing is they can resize _up_ but if they resize down and then back up weird things happen. Is that correct?

Reply:
Yeah that's the problem but I don't even think we need to be that specific with the warning, we should just be like 'resizing won't work right below this version'

Reply:
Like it won't be long before we're healthily several versions past that cutoff date and this becomes moot.

Reply:
> we should just be like 'resizing won't work right below this version'

![luke-skywalker](https://github.com/mozilla/experimenter/assets/4909307/c98b6582-c8b4-49da-ab8f-4e41a8b4578e)